docs/FRSKY_R9.md

```diff
@@ -54,24 +54,30 @@ If you have the older 2018, "ACCST" version of the R9M, you will want to perform
+The mLRS git repository includes an Arduino sketch which allows several supported ESP32 boards to be used as a wireless bridge to connect the serial port to any of the many available GCS such as Mission Planner or QGroundControl. This approach can also eliminate the need for a separate inverter dongle.  Two of these boards, the M5Stamp Pico Mate and the M5Stamp C3U Mate from M5Stack allow pin layouts which are especially convenient to connect directly to the serial pins on the R9M Tx module. The [M5Stamp C3U Mate](https://shop.m5stack.com/collections/m5-controllers/products/m5stamp-c3u-mate-with-pin-headers) is the easiest option as it can be flashed via its included USB port rather than requiring a separate programmer, but it doesn't support the Bluetooth Serial Port Profile.  The M5Stamp Pico Mate is the best choice if you might want to use Bluetooth serial.  Both of these 2.4 GHz wireless bridge options work especially well with 900 MHz systems like the R9 since the separate frequency range minimizes interference.
 
 To install the sketch on the M5Stamp C3U Mate, use the Arduino IDE. Open the mlrs-wireless-bridge.ino sketch from the mLRS esp/mlrs-wireless-bridge folder, edit the mlrs-wireless-bridge.ino file to uncomment only the MODULE\_M5STAMP\_C3U\_MATE\_FOR\_FRSKY\_R9M define, select the ESP32C3 Dev board in the IDE, connect the M5Stamp C3U Mate module USB connector to your computer while holding down the center button, and upload the sketch via the IDE.
 
 __Be sure to unplug the M5Stamp C3U Mate from the back of the R9M when programming via USB to avoid feeding 5 volt power back to R9M which might cause damage.__
 
 <img src="images/Frsky_R9M_M5Stamp_C3U_installed.jpg" width="360px">
 
-Connecting the M5Stamp C3U Mate to the R9M is easy: Remove the screw and the plastic cover. Cut a 5 pin length of the included pin header female connector and pull out the pin from the last position. This position serves as a key to avoid plugging in the board incorrectly. Solder the pins in the thru holes as shown below (key position hanging over the left) and reinstall the plastic cover.  After programming the sketch, install on the back of the R9M as shown above. Then, you can connect your GCS computer or mobile device to the "mLRS AP UDP" WiFi access point and connect the GCS via UDP on port 14550.
+Connecting the M5Stamp C3U or Pico Mate to the R9M is easy: Remove the screw and the plastic cover. Cut a 5 pin length of the included pin header female connector and pull out the pin from the last position. This position serves as a key to avoid plugging in the board incorrectly. Solder the pins in the thru holes as shown below (key position hanging over the left) and reinstall the plastic cover.  After programming the sketch, install on the back of the R9M as shown above. Then, you can connect your GCS computer or mobile device to the "mLRS AP UDP" WiFi access point and connect the GCS via UDP on port 14550.
```

Comment:
- maybe better "... M5Stamp C3U Mate or Pico Mate ..."
- I find it a bit unclear whose screw and plastic cover to remove, mabye "... Remove the M5Stamp's screw..." 
- the paragraph befroe also needs taking account for both species now ...

Reply:
I'd rather not provide a link to the Pico Mate without the programmer.  I want to encourage new users to take the easy route and reduce support issues.  I'll update the other wording you mentioned.

Reply:
I added a duplicate link to the kit in the latest commit